source/_docs/guides/lando.md

```diff
@@ -0,0 +1,165 @@
+</div>
+
+Create a separate folder to hold your Lando-powered installations rather than mixing with installs managed by other server environments.
+
+## Installing WordPress Locally
+
+Getting started with Lando can be disorienting at first. Fear not, mighty developer, that’s why this documentation exists! We’ll begin by installing and spinning up a fresh WordPress without Pantheon integration.
+
+1. Open your terminal and get to the directory where you plan to run your fancy Lando WP site.
+1. While in your terminal simply type `lando init`.
+
+![Terminal command for initialiting a new lando site install][image-2]
+
+	You may choose to start from your current directory or you can clone from Pantheon, Github, or another remote repository, or zip file. For this step, we’ll choose Current Working Directory. 
+	
+
+1. Next, you’ll choose WordPress as your starting recipe.
+
+![Choose your Lando recipe to spin up a new website.][image-3]
+
+1.  For your webroot, you may press enter and Lando will spin up the site from your current directory.
+
+1.  Enter the name of the site. Use something you’ll remember!
+
+Your local site has been created but your still not done! You still need to start the app, download WordPress and then install WordPress. Sure, it’s a few steps but the command line practice is good for bone growth and keyboard-punching-dexterity.
+
+1. In your terminal type `lando start`. This starts your lando app and gives you some basic information like your Appserver URLs to access the site in your local browser of choice.
+
+![alt text][image-4]
+
+Then type `lando wp core download`. This will download the core files for your website. This step isn’t necessary if you started with a Git or Pantheon repository instead of the current working directory.
+
+<div class="alert alert-info" role="alert" markdown="1">
+#### Note {.info}
+This is where we talk a little bit about WP-CLI, Lando, and You. To use WP-CLI commands in Lando, simply prefix your wp command with lando and you’re good to go, Lando will run your WP-CLI command all nice like. 
+</div>
+
+1. The `lando wp core download` command downloads the current stable release of WordPress and unpacks it in your current working directory.
+
+1. Your WP configuration file still needs to be created. You can actually do this from the command line. However, we’re going to go to do this the ol’ fashioned way through the browser.
+
+1. In your browser, enter in the URL given to you above: _http://yourappname.lndo.site_. You’ll notice WordPress’s _Famous Five Minute install_ screen.
+
+1. Click next after choosing your preferred language and click Let’s Go on the following screen.
+
+1. In Lando, for a WordPress installation without Pantheon, the Database, Username, and Password are all `wordpress` and Database Host is `database` (the all values are case-sensitive). Enter your credentials and complete the installation process! Congrats! You now have Lando installed with WordPress!
```

Comment:
Using these credentials, I'm unable to establish a connection to the database.

![image](https://user-images.githubusercontent.com/2349184/65340245-6e606f80-db93-11e9-91e2-a5bc7c0cac04.png)

I haven't found anything in Lando's docs to explain it. @dustinleblanc do you have any input?

Reply:
Dustin and I figured it out, error on my par.